tests/test_standard_loader_integration.py

```diff
@@ -130,28 +130,25 @@ def _test_gen3_loading(self, test_json, bundle_guid, file_guid, file_version, mo
             self._load_file(tmp_json, more_args=more_args)
 
             message("Wait for newly loaded bundle to appear in search results")
-            search_results = self._search_for_bundle(bundle_guid)
+            bundle = self._get_bundle(bundle_guid)
```

Comment:
Why did you make this change? The purpose of searching for the bundle is to ensure that that newly loaded bundle was successfully indexed, which is part of the loading process.
This changes you propose here are rejected.

Reply:
I didn't realize why searching was important. Now that we are using auth, the bundles need to be accessed this way, but can be searched for as well to make sure they are indexed.